src/lib.rs

```diff
@@ -10,20 +10,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![deny(warnings)]
```

Comment:
According to: https://github.com/rust-unofficial/patterns/blob/master/anti_patterns/deny-warnings.md this is an anti-pattern. It doesn't really bother me though. If it becomes a problem in the future with newer versions of rustc it's simple enough to remove though.